README.md

```diff
@@ -1 +1,73 @@
 # ArDrive Price Calculator
+
+## Developer Setup
+
+Follow these steps to get the developer environment up and running:
+
+### Install Yarn 2
+
+The ArDrive Price Calculator uses Yarn 2, so install the latest version with the [yarn installation instructions][yarn-install]. In most cases:
+
+```shell
+# Brew:
+brew install yarn
+
+# Or with NPM:
+npm install -g yarn
+```
+
+We also use husky. To enable hooks locally, you will need to run:
```

Comment:
Nit: explain to the reader why we use husky for and how we're using it (e.g. if there's any tests we run on pre-commit, etc.)

Reply:
Added some info, and also added that `yarn test` to the pre-commit hook. If a test fails, husky rejects the commit